src/modules/back-to-top/back-to-top.component.spec.ts

```diff
@@ -53,12 +53,12 @@ describe('StacheBackToTopComponent', () => {
 
   it('should show when the window y offset is greater than the specified offset', fakeAsync(() => {
     component.offset = 0;
-    TestUtility.fireDomEvent(windowRef, 'scroll');
+    TestUtility.triggerDomEvent(windowRef, 'scroll');
```

Comment:
Did they rename that? Or do you like trigger better? Lol. I don't mind either way.